packages/graphql/src/emitter.ts

```diff
@@ -1,11 +1,37 @@
-import type { EmitContext } from "@typespec/compiler";
-import { emitFile, resolvePath } from "@typespec/compiler";
-
-export async function $onEmit(context: EmitContext) {
-  if (!context.program.compilerOptions.noEmit) {
-    await emitFile(context.program, {
-      path: resolvePath(context.emitterOutputDir, "output.txt"),
-      content: "Hello world\n",
-    });
-  }
+import type { EmitContext, NewLine } from "@typespec/compiler";
+import { resolvePath } from "@typespec/compiler";
+import type { GraphQLEmitterOptions } from "./lib.js";
+import { createGraphQLEmitter } from "./schema-emitter.js";
+
+const defaultOptions = {
+  "new-line": "lf",
+  "omit-unreachable-types": false,
+  "strict-emit": false,
```

Comment:
would just `strict` be maybe better?

Reply:
Yeah, it could be. I just wasn't sure if we'll have something like `strict-operation` or `strict-<something-else>`, so I kept this as `strict-emit` to denote that this applies to the emitter holistically.

Reply:
i think even if you did, `strict: true` being the overall setting that enables all strict system likes typescript config does make sense.

It also maybe to consider that this should instead be a linting ruleset

```yaml
linter:
   extends:
     - "@typespec/graphql/strict"
```

Reply:
done